android/app/build.gradle

```diff
@@ -135,6 +135,7 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-photo-view')
```

Comment:
Ah okay yea pinch to zoom is nice, and that makes sense! Either way works for me :)